projects/storefrontapp-e2e-cypress/cypress.config.ci.ts

```diff
@@ -29,6 +29,8 @@ export default defineConfig({
     OCC_PREFIX: '/occ/v2',
     OCC_PREFIX_USER_ENDPOINT: 'users',
     OCC_PREFIX_ORDER_ENDPOINT: 'orders',
+    mail_CCV2_URL: 'http://mail-ccv2.westeurope.azurecontainer.io:8025',
+    mail_CCV2_PREFIX: '/api/v2',
```

Comment:
Sorry for nitpicking. Can we use capitalise letters for such properties to unify with the rest?

Reply:
don't worry, it's my fault.....    have changed